bin/ruby-build

```diff
@@ -91,7 +91,7 @@ os_information() {
   if type -p lsb_release >/dev/null; then
     lsb_release -sir | xargs echo
   elif type -p sw_vers >/dev/null; then
-    echo "OS X $(sw_vers -productVersion)"
+    echo "$(sw_vers -productName) $(sw_vers -productVersion)"
```

Comment:
What's return value with `sw_vers -productName` on Big Sur?

Reply:
https://twitter.com/sleeve/status/1276191458714775552
https://twitter.com/never_released/status/1275444907394490373

it seems on big sur, it will return `macOS`

Reply:
got it. It seems reasonable.